### DISCUSSION TITLE
v2/zh/guide/feature/photo-swipe

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#377] [Announcements]
# v2/zh/guide/feature/photo-swipe

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/feature/photo-swipe.html

**Comment by VivaAlexYu** [comment]
It is recommended to add that when the mouse moves over the picture, the mouse arrow will change to zoom in on the picture, so as to better remind the user that they can view it at a larger size

**Reply by Mister-Hope** [reply]
Added https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/ee2fc793025170d2603ac48c6a8c4d0b25beac6b.

Also, chinese are recommanded under zh docs.
